### ISSUE TITLE
EtherCalc- Name of the person making changes to data fails to appear when more than 1 user editing at a same time

### ISSUE THREAD

**Issue by Rn-jit** [audreyt/ethercalc#749]
1) Suppose there are two users User A and User B.
2) As user A create a spreadsheet and enter some data.
3) As user B access the same spreadsheet and edit the data at the same time.

Name of the person making changes to data fails to appear when more than 1 user editing at a same time.

Is it possible to show the name of the person who are editing the sheet at the same time?

**Comment by eddyparkinson** [comment]
yes. Google sheets has something close to this.

To give actual names, you would need a login system.   There have been several request to have a login feature - We use express, there are existing  login systems that work with express. 

The sandstorm version has login.

**Comment by Rn-jit** [comment]
Yes I have login system. And the sandstorm version also has login system, still there is no such facility that match to my requirement. It means I can't see the  name of the person who actually edits the sheet.

I have a use case.
I have a web app with login system. Here all the team mates are connected and communicates with each other and also share the sheet. How I came to know who is editing the sheet. The name of the person not showing. How would I reflect it ?

**Comment by patlachance** [comment]
@Rn-jit could you please elaborate on how you implemented your login system? Thanks!
